packages/oscal-react-library/src/components/OSCALMetadata.tsx

```diff
@@ -306,34 +308,72 @@ const MetadataInfoList: React.FC<MetadataInfoListProps> = (props) => {
   );
 };
 
-export interface OSCALMetadataPartyProps {
+interface OSCALMetadataPartyLogoProps extends ResolvableLinkHref {
+  link: OSCALLink;
+  partyName?: string;
+}
+
+const OSCALMetadataPartyLogo: React.FC<OSCALMetadataPartyLogoProps> = ({
+  link,
+  backMatter,
+  parentUrl,
+  partyName,
+}) => {
+  let logoUri = undefined;
+
+  if (backMatter) {
+    try {
+      logoUri = resolveLinkHref(backMatter, link.href, parentUrl, /^image\//, false);
+    } catch (err) {
+      // Silently fail on unresolved diagram resources
+    }
+  }
+
+  const imageSrc = logoUri ?? link.href;
```

Comment:
Im not sure if this is the cleanest way to deal with `resolveLinkHref` failing and the fact that `backMatter` doesn't need to exisit.

I was thinking that it could be `resolveLinkHref` that deals with an undefined `backMatter` but just did this for now.